File: java/src/main/java/co/edu/sena/ejercicios/ejercicio11/Parte.java
package co.edu.sena.ejercicios.ejercicio11;

public class Parte {
    private String serial;
    private String marca;

    public Parte(String serial, String marca) {
        this.serial = serial;
        this.marca = marca;
    }

    public String getSerial() {
        return serial;
    }

    public void setSerial(String serial) {
        this.serial = serial;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }
}

File: typescript/src/ejercicio11/parte.ts
export class Parte {
    private _serial: string;
    private _marca: string;


    constructor(serial: string, marca: string) {
        this._serial = serial;
        this._marca = marca;
    }


    get serial(): string {
        return this._serial;
    }

    set serial(value: string) {
        this._serial = value;
    }

    get marca(): string {
        return this._marca;
    }

    set marca(value: string) {
        this._marca = value;
    }
}